From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 31 Mar 2020 22:09:38 +0000
Subject: extensions-app: Add category in .desktop file

Predefined categories aren't a great way for organizing installed
applications, but they have their use in "stores" like Software
or flathub.

Not listing any category means we fall through the cracks, so
pick the (hopefully) least inappropriate one ...

https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/1160

(cherry picked from commit 360f5b16421d026715c9c57cc4af6ea6e2749210)

Origin: upstream, 3.36.2, commit:aebfab72071e9b7fbfa9478cc8304e677377c266
---
 subprojects/extensions-app/data/org.gnome.Extensions.desktop.in.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/subprojects/extensions-app/data/org.gnome.Extensions.desktop.in.in b/subprojects/extensions-app/data/org.gnome.Extensions.desktop.in.in
index 588e68b..a935780 100644
--- a/subprojects/extensions-app/data/org.gnome.Extensions.desktop.in.in
+++ b/subprojects/extensions-app/data/org.gnome.Extensions.desktop.in.in
@@ -6,5 +6,5 @@ Icon=@app_id@
 Comment=Configure GNOME Shell Extensions
 Exec=@bindir@/@prgname@
 DBusActivatable=true
-Categories=GNOME;GTK;
+Categories=GNOME;GTK;Utility;
 OnlyShowIn=GNOME;
